qt/scientific_interfaces/test/ISISReflectometry/Reduction/ValidateRowTest.h

```diff
@@ -211,9 +237,9 @@ class ValidateRowTest : public CxxTest::TestSuite {
   }
 
   void testFailsForInvalidFirstAndSecondTransmissionRun() {
-    TS_ASSERT_EQUALS(
-        std::vector<int>({0, 1}),
-        boost::get<std::vector<int>>(parseTransmissionRuns("1,000", "10ABSC")));
+    TS_ASSERT_EQUALS(std::vector<int>({0, 1}),
```

Comment:
( and { on vector

Reply:
I remember why this is now - the parentheses are required because otherwise the commas get interpreted as argument separators for the macro rather than as part of the vector initialisation, so it doesn't compile